modules/virt-reviewing-top-consumers.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc
+
+[id="virt-reviewing-top-consumers_{context}"]
+= Reviewing top consumers
+
+In the *Administrator* perspective, you can view the {VirtProductName} dashboard where top consumers of resources are displayed.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the *Administrator* perspective in the {VirtProductName} web console, navigate to *Observe* -> *Dashboards*.
+
+. Select the `KubeVirt/Infrastructure Resources/Top Consumers` dashboard from the *Dashboard* list.
```

Comment:
Could `KubeVirt/Infrastructure Resources/Top Consumers` also just be marked up with asterisks (instead of backticks) like the rest of the UI elements?

Reply:
Yep. Done.